Slicing_indexing1.py

```diff
@@ -0,0 +1,17 @@
+a = (len(h1)) % 2
```

Comment:
Programs results in error.

<img width="910" alt="image" src="https://user-images.githubusercontent.com/38807024/203931973-ad864b0a-33de-498a-9d6d-194f4bdbf6f4.png">